samples/DLLMethodCaller/DLLMethodCallerEA.mq4

```diff
@@ -41,36 +41,59 @@
 	int nquotes_get_property_array_size(string name);
 	int nquotes_get_property_adouble(string name, double& value[]);
 #import
+//extern int MoneyAmount = 12345;
+//extern datetime MyBirthday;
+//extern bool AmMarried;
 
 int init()
 {
+   EventSetTimer(5);
```

Comment:
why this is needed?

Reply:
This allows you to not wait on ticks if market is closed. It just triggers operations without dependency on market being open.